_data/task.yml

```diff
@@ -37,6 +37,14 @@ websites:
       twitter: fruux
       tfa: No
 
+    - name: Microsoft To-Do
+      url: https://todo.microsoft.com/
+      img: microsofttodo.png
+      tfa: Yes
+      sms: Yes
+      software: Yes
+      doc: https://windows.microsoft.com/en-us/windows/two-step-verification-faq
```

Comment:
`https://windows.microsoft.com/en-us/windows/two-step-verification-faq` redirects to `https://support.microsoft.com/en-us/help/12408/microsoft-account-about-two-step-verification`. Please use the latter.

Reply:
Should be fixed now.